lib/takeaway.rb

```diff
@@ -0,0 +1,23 @@
+class Takeaway
+
+  attr_reader :name
+
+  def initialize(name)
+    # Not enforced, but menu hash structure is {"dish name" => cost}
+    @menu = {}
+    @name = name
+  end
+
+  def add_menu_items(items)
+    @menu.merge!(items)
+  end
+
+  def list_menu
```

Comment:
It's dangerous to expose internal implementations like this - I could hack around with the hash and break the program.  It's less of an argument if you are not sanitizing the input in `add_menu_items`, but it's still something to consider.  You could return a copy of the hash instead to prevent accidental changes.